rotkehlchen/tests/fixtures/globaldb.py

```diff
@@ -81,6 +81,16 @@ def create_globaldb(
     return handler
 
 
+def add_unique_cache_table(globaldb) -> None:
```

Comment:
I checked the usages of this method and, as described in the database fixture, I don't see any point in having it. Instead of adding the table dynamically, `rotkehlchen/data/global.db ` file should be updated instead and contain the new table